resources/docs/simulation/creating-simulations/collaboration/README.md

```diff
@@ -27,7 +27,7 @@ Should you choose to allow it, projects in [hCore](/platform/core) can be shared
 
 To generate a share link, click the 'Share' button in the top-right hand corner of HASH. [Read more about sharing simulations &gt;](/docs/simulation/creating-simulations/collaboration/sharing-releasing)
 
-## Coming soon...
+## Coming soon
```

Comment:
@akash-joshi What rule results in this? Interesting

Reply:
This is the rule which is causing it: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md026---trailing-punctuation-in-heading

Let me know if we want to disable it.

Also, thanks for going through all the changes! I know there are a lot.

Reply:
We can probably disable this rule for now. Having `!`, `...` or `?` at the end of the headers is fine (but we may want to disable `;` `,` etc. in the future when we have more time to thing of the rules)